classes/Tools.php

```diff
@@ -849,8 +849,6 @@ public static function displayPriceSmarty($params, &$smarty)
     /**
      * Return price converted.
      *
-     * @deprecated since 1.7.4 use convertPriceToCurrency()
```

Comment:
👍 nice, thanks

for people who dont understand this: see https://github.com/PrestaShop/PrestaShop/issues/14956

Reply:
You're right, I found the original commit: https://github.com/PrestaShop/PrestaShop/pull/8411/files#diff-9079f61ba1c5da7f9f142723d5d42ea0R683